curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-regular-expressions-by-building-a-spam-filter/64221007887f38213fa57827.md

```diff
@@ -0,0 +1,195 @@
+---
+id: 64221007887f38213fa57827
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+One last thing with this expression. You don't actually need the match value from your capture group, so you can turn it into a <dfn>non-capturing group</dfn>. This will allow you to group the characters together without preserving the result.
+
+To turn a group into a non-capturing group, you add `?:` after the opening parenthesis. For example, `(?:a|b)` will match `a` or `b`, but will not capture the result.
```

Comment:
```suggestion
To create a non-capturing group in a regular expression, you can add `?:` after the opening parenthesis of a group. For instance, `(?:a|b)` will match either `a` or `b`, but it will not capture the result.
```